apps/condo/domains/user/schema/RegisterUserExternalIdentityService.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * Generated by `createservice user.RegisterUserExternalIdentityService`
+ */
+
+const { GQLCustomSchema, getById } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/user/access/RegisterUserExternalIdentityService')
+const { UserExternalIdentity } = require('@condo/domains/user/utils/serverSchema')
+const { IDP_TYPES } = require('@condo/domains/user/constants/common')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { DV_VERSION_MISMATCH } = require('@condo/domains/common/constants/errors')
+const {
+    USER_NOT_FOUND,
+    EMPTY_EXTERNAL_IDENTITY_ID_VALUE,
+} = require('../constants/errors')
+
+const errors = {
+    DV_VERSION_MISMATCH: {
+        mutation: 'registerUserExternalIdentity',
+        variable: ['data', 'dv'],
+        code: BAD_USER_INPUT,
+        type: DV_VERSION_MISMATCH,
+        message: 'Unsupported value for dv',
+    },
+    USER_NOT_FOUND: {
+        mutation: 'registerUserExternalIdentity',
+        variable: ['data', 'user', 'id'],
+        code: BAD_USER_INPUT,
+        type: USER_NOT_FOUND,
+        message: 'Could not find User by provided id',
+    },
+    EMPTY_EXTERNAL_IDENTITY_ID_VALUE: {
+        mutation: 'registerUserExternalIdentity',
+        variable: ['data', 'identityId'],
+        code: BAD_USER_INPUT,
+        type: EMPTY_EXTERNAL_IDENTITY_ID_VALUE,
+        message: 'Can not create UserExternalIdentity for empty identityId',
+    },
+}
+
+const RegisterUserExternalIdentityService = new GQLCustomSchema('RegisterUserExternalIdentityService', {
+    types: [
+        {
+            access: true,
+            type: `enum IdentityType { ${IDP_TYPES.join(' ')} }`,
+        },
+        {
+            access: true,
+            type: 'input RegisterUserExternalIdentityInput { dv: Int! sender: SenderFieldInput! user: UserWhereUniqueInput! identityId: String! identityType: IdentityType! meta: JSON! }',
+        },
+        {
+            access: true,
+            type: 'type RegisterUserExternalIdentityOutput { status: String!, id: String! }',
+        },
+    ],
+
+    mutations: [
+        {
+            access: access.canRegister,
+            schema: 'registerUserExternalIdentity(data: RegisterUserExternalIdentityInput!): RegisterUserExternalIdentityOutput',
```

Comment:
Why you need a special mutation for that? You can use createUserExternalIdentityService

Reply:
There is no any BL inside this mutation

Reply:
I supposed to use RegisterUserExternalIdentityService to create UserExternalIdenty. Now it doesn't have specific business logic in here for now. But once we will integrate it with sbbol/sberId this service will be a place for business logic.